vmslave-config/pipeline/group_vars/deb

```diff
@@ -1,2 +1,2 @@
 # on-imagebuilder is not working in ansible more than 2.0
-slave_ansible_version: "{{ lookup('env','SLAVE_ANSIBLE_VERSION') }}"
\ No newline at end of file
+ansible_version: "{{ lookup('env','DEB_ANSIBLE_VERSION') }}"
```

Comment:
I confused by the (re)naming being done. It looks like the ENV variable being used is now DEB_ANSIBLE_VERSION (which makes sense). "slave_ansible_version" is also changed to just "ansible_version", though the checks for slave_ansible_version being set were left in vmslave-config/ansible/roles/install-ansible/tasks/main.yml...

I can't find any other reference to "slave_ansible_version", but should those checks be removed?

(also: missing newline ;) )